problemi/carnot/carnot.cpp

```diff
@@ -0,0 +1,168 @@
+#define DOCTEST_CONFIG_IMPLEMENT_WITH_MAIN
+
+#include <cmath>
+#include <iomanip>
+#include <iostream>
+#include <numbers>
+
+#include "doctest.h"
+
+constexpr double R{8.314};
+constexpr double gammaMono{5. / 3};  // Per un gas monoatomico
+
+class State {
+  double V_;
+  double T_;
+  double P_;
+
+ public:
+  State(double V, double T)
+      : V_(V), T_(T), P_((R * T) / V) {}  // Costruttore di stato gas "ideale".
```

Comment:
```suggestion
      : V_(V), T_(T), P_((R * T) / V) {}
```
se il commento e' veramente necessario, e' meglio metterlo non in fondo alla riga, altrimenti la formattazione soffre